test_runtime.sh

```diff
@@ -79,8 +79,17 @@ cp runtimetest ${TESTDIR}
 
 oci-runtime-tool generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
 
-TESTCMD="${RUNTIME} start $(uuidgen)"
+conID=$(uuidgen)
+
+CREATECMD="${RUNTIME} create ${conID}"
+TESTCMD="${RUNTIME} start ${conID}"
 pushd $TESTDIR > /dev/null
+if ! ${CREATECMD}; then
+	info "Runtime is ${RUNTIME}, no need to create before start"
+else
+	info "Runtime is ${RUNTIME}, need to create before start"
+fi
```

Comment:
when executed "${RUNTIME} create ${conID}", you can use `$?` to get the exit code.
just as:
`# sleep 1
# echo $?
0`